File: rb/utility/prepare_body.rb
# PassantenfrequenzStadtStgallen SDK utility: prepare_body
module PassantenfrequenzStadtStgallenUtilities
  PrepareBody = ->(ctx) {
    ctx.op.input == "data" ? ctx.utility.transform_request.call(ctx) : nil
  }
end

File: php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// PassantenfrequenzStadtStgallen SDK utility: prepare_body

class PassantenfrequenzStadtStgallenPrepareBody
{
    public static function call(PassantenfrequenzStadtStgallenContext $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}
